pkg/features/kube_features.go

```diff
@@ -58,6 +58,16 @@ const (
 	// Enables usage of any object for volume data source in PVCs
 	AnyVolumeDataSource featuregate.Feature = "AnyVolumeDataSource"
 
+	// owner: @MikeSpreitzer, @tkashem, @linxiulei
+	// beta: 1.33
+	//
+	// Make API Priority and Fairness use modern configuration, which
+	// differs from the old in these ways:
+	// - introduce priority level and flow schema for events;
+	// - generally reorganize to stop working around lack of borrowing;
+	// - increase the nominal concurrency shares for leader election.
+	APFNewConfig featuregate.Feature = "APFNewConifg"
```

Comment:
What if we will decide to change APF config again in the future? 
Maybe call it APFv133Config or sth?

Reply:
I assume that the form of growth will be additional features, since a feature is defined to be a boolean choice.

Reply:
I called it "APFv134Config", since release 1.33 enhancements are already frozen.